apps/style/common.scss

```diff
@@ -1558,13 +1558,61 @@ $see-examples-thick-underline-color: $yellow;
   max-width: 100% !important;
 }
 
-a.WireframeSendToPhone_send-to-phone-link {
+.WireframeButtons_container {
+  display: flex;
```

Comment:
While IE11 does have a large number of flexbox-related bugs, they're mostly vertical layout problems that don't come into play here.  See the "Known Issues" tab on CanIUse or [the flexbugs repo](https://github.com/philipwalton/flexbugs).  Thanks for calling this out though - we'll make sure and verify on IE11.